src/main/kotlin/eu/kevin/api/models/auth/authentication/request/StartAuthenticationRequest.kt

```diff
@@ -3,6 +3,7 @@ package eu.kevin.api.models.auth.authentication.request
 data class StartAuthenticationRequest @JvmOverloads constructor(
     val requestId: String,
     val redirectUrl: String,
+    var webhookUrl: String? = null, // Needed just if AuthenticationScopes.PAYMENT_POS is present
```

Comment:
Instead of writing inline comments, class should be fully documented. There will be a separate task for this, so just remove this comment.

Reply:
Comment removed